crates/host/src/wasmbus/config.rs

```diff
@@ -1,21 +1,27 @@
 use serde::{Deserialize, Serialize};
-use url::Url;
 
+#[allow(clippy::module_name_repetitions)]
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 /// wasmCloud Host configuration
 pub struct Host {
-    /// URL to connect to
-    pub url: Url,
-    /// Optional host seed
+    /// NATS server host to connect to
```

Comment:
I think internally, this should not be split into "host" and "port", but be just a single URL.

Apart from simplifying usage internally, a more important aspect of this is the scheme - while I understand that in current OTP implementation it is not possible to force a particular scheme to be used, a developer should be able to e.g. enforce a connection to NATS via TLS, which can only be done by specifying the scheme

Reply:
Can we revisit this later? I mirrored the options we [use in wash](https://github.com/wasmCloud/wash/blob/b0e746be713d070b4400294ec401b87444bd5741/src/up/mod.rs#L94), and this will make it easier to maintain backwards compatibility with tooling that starts hosts. (Whether we use a single URL or a host + port, we'll need to refactor anyway to split that into 3 sets of args: `WASMCLOUD_CTL_*`, `WASMCLOUD_RPC_*`, and `WASMCLOUD_PROV_RPC_*`)

Reply:
Well, we can, but that would mean we have to revert this particular change, since we already are using a URL here, which to me seems as unnecessary churn.

Note, that I'm not asking to change the binary arguments and/or environment variables, just keeping the type of the field on the struct the same as it already was.